### DISCUSSION TITLE
Custom control bar with own layout?

### DISCUSSION THREAD

**Discussion by Firehold** [videojs/video.js#7341] [General]
Hello,
i saw that it is possible to customize the controlbar and style it as you want.
Is it also possible to create a complete own layout so i dont have a bar but buttons on given positions for the controls?

Thank you very much in advance

**Comment by gkatsev** [comment]
Yup, you can style the control bar using CSS. See the themes blogpost https://mux.com/blog/the-new-video-js-themes/ and the themes repo (https://github.com/videojs/themes) for more ideas.

You can also create your own UI. Make sure to not set `controls` on the embed or set `controls: false` as an option to turn off the built-in controls. You can then use the [player API](https://docs.videojs.com/player) to interact with the player. Internally, the controls use the player API, there isn't really much magic happening there.
